zproject/computed_settings.py

```diff
@@ -110,14 +110,7 @@
 # This is overridden in test_settings.py for the test suites
 BAN_CONSOLE_OUTPUT = False
 
-# Google Compute Engine has an /etc/boto.cfg that is "nicely
-# configured" to work with GCE's storage service.  However, their
-# configuration is super aggressive broken, in that it means importing
-# boto in a virtualenv that doesn't contain the GCE tools crashes.
-#
-# By using our own path for BOTO_CONFIG, we can cause boto to not
-# process /etc/boto.cfg.
-os.environ['BOTO_CONFIG'] = '/etc/zulip/boto.cfg'
```

Comment:
Just to be explicit, is the theory that we get to remove the hack here because the BOTO_CONFIG setting that GCE sets doesn't do anything?

If you have a bit of time, you may want to create a free GCE account and test out what happens here.